flink-examples/flink-examples-batch/src/main/java/org/apache/flink/examples/java/clustering/KMeans.java

```diff
@@ -291,31 +297,54 @@ public Centroid map(Tuple3<Integer, Point, Long> value) {
 	private static String centersPath = null;
 	private static String outputPath = null;
 	private static int numIterations = 10;
-	
-	private static boolean parseParameters(String[] programArguments) {
-		
-		if(programArguments.length > 0) {
-			// parse input arguments
+
+	private static final Option POINTS_PATH_OPTION =
+		new Option("points").alt("P").help("The path to the input points");
+	private static final Option CENTERS_PATH_OPTION =
+		new Option("centroids").alt("C").help("The path to the input centroids");
+	private static final Option OUTPUT_PATH_OPTION =
+		new Option("output").alt("O").help("The path where the output will be written");
+	private static final Option NUM_ITERATIONS_OPTION =
+		new Option("iterations").alt("I").help("The number of iteration performed by the K-Means algorithm");
+
+	private static boolean parseParameters(final ParameterTool params) throws RequiredParametersException {
```

Comment:
I thought about it but was afraid of polluting the ´main´ method with "secondary" concerns. I'm going to fix this right now. I'll keep this in mind for other examples as well, thanks!